torch/autograd/profiler.py

```diff
@@ -202,15 +202,15 @@ class profile(object):
             Default: ``False``
 
         record_shapes (bool, optional): If shapes recording is set, information
-        about input dimensions will be collected. This allows one to see which
-        dimensions have been used under the hood and further group by them
-        using prof.key_averages(group_by_input_shape=True). Please note that
-        shape recording might skew your profiling data. It is recommended to
-        use separate runs with and without shape recording to validate the timing.
-        Most likely the skew will be negligible for bottom most events (in a case
-        of nested function calls). But for higher level functions the total
-        self cpu time might be artificially increased because of the shape
-        collection.
+            about input dimensions will be collected. This allows one to see which
```

Comment:
did you mean to reindent this? lol

Reply:
I think you did!

Reply:
Aye, i noticed the formatting for the `record_shapes` arg was...unexpected :P https://pytorch.org/docs/master/autograd.html#torch.autograd.profiler.profile